server/repositories/WalletRepository.js

```diff
@@ -0,0 +1,31 @@
+/* 
+ * The model for: entity, wallet, entity table and so on
+ */
+const HttpError = require("../utils/HttpError");
+const knex = require('../database/knex');
+const expect = require("expect-runtime");
+
+class WalletRepository {
+
+  constructor(){
+  }
```

Comment:
I don't think we need this constructor (and also in TokenRepository)

Reply:
You are right, let's remove it away.